sos/report/plugins/ceph_common.py

```diff
@@ -95,12 +95,22 @@ def setup(self):
                     "/var/snap/microceph/common/logs/ceph.audit.log*",
                 ])
 
-            self.add_cmd_output("microceph status", subdir="microceph")
             self.add_cmd_output("snap info microceph", subdir="microceph")
-            # Collect all but exclude any keyrings
-            self.add_cmd_output("microceph cluster sql 'select * from config \
-                                 where key NOT LIKE \"%keyring%\"'",
-                                subdir="microceph")
+
+            cmds = [
+                'client config list',
+                'cluster list',
+                'cluster config list',
+                # exclude keyrings from the config db
+                'cluster sql \'select * from config where key NOT LIKE \
+                    \"%keyring%\"\'',
+                'disk list',
+                'log get-level',
+                'status'
```

Comment:
lgtm

minor nit pick, could add a comma at the  end :). When we add new commands, we just add new commands on new line

Reply:
ack, done